projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-ai-feature/index.ts

```diff
@@ -22,14 +22,18 @@ export default function useAiFeature() {
 			usagePeriod,
 			requestsCount: allTimeRequestsCount,
 			requestsLimit: freeRequestsLimit,
+			tierPlansEnabled,
 		} = featureData;
 
 		const planType = getPlanType( currentTier );
 
+		// TODO: mind this hardcoded value (3000),
+		// maybe provide it from the backend but we'd be replacing the 9 Billion limit with 3k
+		const currentTierLimit = tierPlansEnabled ? currentTier?.limit || freeRequestsLimit : 3000;
```

Comment:
I think we don't need to worry about the fair usage limiting here, for two reasons:

- since tiered plans will be considered unlimited on the backend (and will migrated sometime in the near future), the currentTierLimit prop will be irrelevant on the UI, since it is ignored for unlimited sites
- having the 3000 value here may bring some issues when we decide to change the number. We already have some discussion going on that may bump it to 12500/month (pe4Cmx-2GI-p2), and we may also decide to adjust it as we move to cheaper models in the future, so having the cap on the plugin code can make things unflexible